op-proposer/proposer/driver.go

```diff
@@ -385,7 +421,11 @@ func (l *L2OutputSubmitter) sendTransaction(ctx context.Context, output *eth.Out
 			return err
 		}
 	} else {
-		data, err := l.ProposeL2OutputTxData(output)
+		consumerId, err := l.ConstructConsumerID(ctx)
+		if err != nil {
+			return err
+		}
+		data, err := l.ProposeL2OutputTxData(output, consumerId)
```

Comment:
this is also called in `ActMakeProposalTx` in op-e2e/actions/l2_proposer.go. 

what's the diff?

Reply:
It is the same logic.